src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,28 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
try to use while here. It will looks like better then `for` loop

Reply:
I knew it!